src/Confluent.Kafka/AdminClient.cs

```diff
@@ -445,6 +462,39 @@ private Task StartPollTask(CancellationToken ct)
                                             }
                                         }
                                         break;
+
+                                    case Librdkafka.EventType.DeleteConsumerGroupOffsets_Result:
+                                        {
+                                            if (errorCode != ErrorCode.NoError)
+                                            {
+                                                Task.Run(() =>
+                                                    ((TaskCompletionSource<List<DeleteConsumerGroupOffsetsResult>>)adminClientResult).TrySetException(
+                                                        new KafkaException(kafkaHandle.CreatePossiblyFatalError(errorCode, errorStr))));
+                                                break;
+                                            }
+
+                                            var result = extractDeleteConsumerGroupOffsetsReports(eventPtr);
+
+                                            if (result.Any(r => r.Error.IsError))
```

Comment:
i think we should throw an exception if any of the partitions had a problem. that would be more analogous to the way other methods are done i think.

Reply:
Makes sense, updated now.